feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -23,8 +24,13 @@ internal class HomeViewModel @Inject constructor(
     private val buildConfigProvider: BuildConfigProvider,
     private val locationProvider: LocationProvider,
     private val gpsSettingsProvider: GpsSettingsProvider,
+    private val appSettingsProvider: AppSettingsProvider,
 ) : ViewModel(),
-    StateDelegate<HomeState> by DefaultStateDelegate(initialState = HomeState()),
+    StateDelegate<HomeState> by DefaultStateDelegate(
+        initialState = HomeState(
+            wasInitialGeolocationPermission = appSettingsProvider.wasInitialGeolocationPermissionRequest().get(),
```

Comment:
Please use datastore + runBlocking. It's acceptable for this case

https://developer.android.com/topic/libraries/architecture/datastore#synchronous

Reply:
Please replace `was` with `is`